client/src/assets/account.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
```

Comment:
There are cleaner ways to do SVGs in React, but this works fine. https://www.sanity.io/guides/import-svg-files-in-react